LayoutTests/imported/w3c/web-platform-tests/html/rendering/the-details-element/details-blockification-expected.txt

```diff
@@ -3,5 +3,5 @@ bar
 foo
 bar
 
-PASS Summary and content should have display:block computed value
+FAIL Summary and content should have display:block computed value assert_equals: expected "block" but got "inline"
```

Comment:
This test relies on `::details-content` to emulate the old behavior of details/summary. It'll re-progress once I implement the pseudo-element: https://bugs.webkit.org/show_bug.cgi?id=275211 (I already have a PR for it)